### PR TITLE
Initial documentation addition

### PR DESCRIPTION
**Pull request by mattfarina**
This is the initial start to some usage docs.

### CODE REVIEW

File: docs/images.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Images
+
+Rancher Desktop provides the ability to build, push, and pull images via the
+[KIM](https://github.com/rancher/kim) project.
```
Comment (sirredbeard) [inline docs/images.md:4]: I think kim is all lowercase

Comment (mattfarina) [reply docs/images.md:4]: That's a good question. KIM is an acronym so it should be KIM. I just went and asked to double check.

File: docs/installing.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Rancher Desktop is delivered as a desktop application. You can download it from
+the [releases page on GitHub](https://github.com/rancher-sandbox/rd/releases).
+
+When run for the first time or when changing versions, Kubernetes container
+images are downloaded. It may take a little time to load on first run for a new
+Kubernetes version.
+
+Note, the Windows installer is not yet signed. This will be coming soon. In the
+meantime, when presenting with a dialog box like the following click on "More
```
Comment (ericpromislow) [inline docs/installing.md:11]: "when presented"

File: docs/installing.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Installing Rancher Desktop
+
+Rancher Desktop is delivered as a desktop application. You can download it from
+the [releases page on GitHub](https://github.com/rancher-sandbox/rd/releases).
+
+When run for the first time or when changing versions, Kubernetes container
+images are downloaded. It may take a little time to load on first run for a new
+Kubernetes version.
+
+Note, the Windows installer is not yet signed. This will be coming soon. In the
```
Comment (ericpromislow) [inline docs/installing.md:10]: I'd say "This will happen soon." but it's a matter of preference

File: docs/installing.md (hunk 2 above)
Comment (ericpromislow) [reply docs/installing.md:11]: "like the following,"  [needs comma]

File: docs/installing.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+meantime, when presenting with a dialog box like the following click on "More
+info".
+
+![](images/windows-notice-1.png)
+
+This will present you with an option to run the application like the following.
```
Comment (ericpromislow) [inline docs/installing.md:16]: "The following dialog will then give you the option to run the application:" ["prefer fewer shorter words, someone must have said at some point"]

**Review by ericpromislow — CHANGES_REQUESTED** [review]
The images are currently 1622x1514 px and are pretty big when I open them. Can we reduce them by 50% or so?

File: docs/installing.md (hunk 2 above)
Comment (mattfarina) [reply docs/installing.md:11]: Fixed

File: docs/installing.md (hunk 4 above)
Comment (mattfarina) [reply docs/installing.md:16]: Fixed

File: docs/installing.md (hunk 3 above)
Comment (mattfarina) [reply docs/installing.md:10]: Updated

**Review by ericpromislow — APPROVED** [review]
LGTM

**Pull request merged**